.shippable.yml

```diff
@@ -47,7 +48,11 @@ build:
           else
             echo "uploading ${TARGET}.tar.bz2 to ${S3_PATH}/";
             pushd build/output;
-            tar --exclude='build.log.bz2' -jcvf ../../${TARGET}.tar.bz2 *-zephyr-*;
+            if [ "${TARGET}" == 'x86_64' ]; then
```

Comment:
Why do we need to special case x86_64 here?

Reply:
Because of the tuple, it is x86_64-multilib-linux-gnu

Reply:
After changing the vendor to zephyr, this is no longer needed.